### DISCUSSION TITLE
Getting a constant load of around 0.1rows/s on freshly created DB with no load

### DISCUSSION THREAD

**Discussion by lakshyasharma14** [planetscale/discussion#111] [Bug Reports] [answered]
I created a new DB and it is showing a constant load of around 0.1rows/s when I haven't connected it or integrated this DB with any application.
I used mysql workbench to connect to the proxy server using pscale CLI tool and created two tables. Then released the development branch into master. Then switched my proxy server to main branch and added 2 rows in each of the tables.
Then I see a constant load of read calls on the DB. The total read calls counter is also increasing.
![image](https://user-images.githubusercontent.com/20574789/134820251-dba60944-b119-4897-9de9-fd71ed017dfe.png)
![image](https://user-images.githubusercontent.com/20574789/134820270-5f87aa1e-500d-4238-be0c-fce7ca49f51b.png)

I tried deleting and creating a fresh DB but the same thing happens with the new one as well.
I see 0 connections active, and also no new queries in query statistics. The query statistics tab only has the queries I executed. I am not sure why is there a constant load of read calls happening from.
![image](https://user-images.githubusercontent.com/20574789/134820406-ad3607c0-eba2-453c-9b6a-0ea58f0ea04f.png)

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @lakshyasharma14,

These reads are coming from health checks we have running on your database. Sorry it's confusing.

We're working on excluding the reads from this check from the row read counts and usage graph on the database dashboard.
